runtime/gc_modron_standard/StandardAccessBarrier.cpp

```diff
@@ -62,6 +62,18 @@ MM_StandardAccessBarrier::newInstance(MM_EnvironmentBase *env)
 	}
 	return barrier;
 }
+void
+MM_StandardAccessBarrier::initializeForNewThread(MM_EnvironmentBase* env)
+{
+	if (_extensions->configuration->isSnapshotAtTheBeginningBarrierEnabled()) {
+
+		_extensions->sATBBarrierRememberedSet->initializeFragment(env, &(((J9VMThread *)env->getLanguageVMThread())->sATBBarrierRememberedSetFragment));
+//TODO: SATB re-enable later
```

Comment:
keep this in, but we'll worry about double barrier in more details later.
for example, we don't have yet the mechanism to enable/disable double barrier.

Reply:
uncommented it.